radar-upload-backend/src/main/java/org/radarbase/upload/api/RecordMapperImpl.kt

```diff
@@ -86,7 +86,7 @@ class RecordMapperImpl(
     override fun fromRecords(records: List<Record>, limit: Int) = RecordContainerDTO(
             limit = limit,
             records = records.map(::fromRecord),
-            lastId = records.sortedBy { record ->  record.id }.last().id!!)
+            lastId = records.maxBy { record ->  record.id!! }?.id!!)
```

Comment:
I think this may still give an error if no records are found. Just
```
records.maxBy { record ->  record.id!! }?.id
```
would probably do, now `null` is allowed.

Reply:
Yes. Already done